x-pack/plugins/observability_solution/infra/public/components/shared/alerts/links/link_to_alerts.tsx

```diff
@@ -10,20 +10,21 @@ import { EuiButtonEmpty } from '@elastic/eui';
 
 export interface LinkToAlertsRuleProps {
   onClick?: () => void;
+  dataTestSubj?: string;
 }
 
-export const LinkToAlertsRule = ({ onClick }: LinkToAlertsRuleProps) => {
+export const LinkToAlertsRule = ({ onClick, dataTestSubj }: LinkToAlertsRuleProps) => {
```

Comment:
I was confused by the `link` term in this component's name. This is not a link it's a button, could we rename it to make it clearer?

```suggestion
export const CreateAlertRuleButton = ({ onClick, dataTestSubj }: LinkToAlertsRuleProps) => {
```

Reply:
💯  let me fix it

Reply:
fixed https://github.com/elastic/kibana/pull/181039/commits/85a45a8bc05d43d4db122ed1b51754ed49b64938